tests/UI/pages/BO/orders/view.js

```diff
@@ -55,6 +55,9 @@ class Order extends BOBasePage {
       page.click(this.UpdateProductButton),
       this.waitForVisibleSelector(page, this.editProductQuantityInput),
     ]);
+    await Promise.all([
+      this.waitForVisibleSelector(page, this.productQuantitySpan(row)),
+    ]);
```

Comment:
```suggestion
    await this.waitForVisibleSelector(page, this.productQuantitySpan(row));
```

No need for `Promise.all`